openapi/components/schemas/Usage.yaml

```
@@ -13,13 +13,13 @@ properties:
     allOf:
       - $ref: ./ResourceId.yaml
   invoiceId:
-    description: ID of the invoice for which usage is considered. This value is added when the invoice with usage is issued.
+    description: ID of the invoice to which usage is applied. This value is populated when the invoice is issued.
     readOnly: true
     nullable: true
     allOf:
       - $ref: ./ResourceId.yaml
   invoiceItemId:
-    description: ID of the invoice item for which usage is considered. This value is added when the invoice with usage is issued.
+    description: ID of the invoice to which usage is applied. This value is populated when the invoice is issued.
```

typo: I missed this in my suggestion

Sorry about that

```suggestion
    description: ID of the invoice item to which usage is applied. This value is populated when the invoice is issued.
```